src/msg/Message.h

```diff
@@ -320,8 +320,8 @@ class Message : public RefCountedObject {
     memset(&footer, 0, sizeof(footer));
   }
 
-  Message *get() {
-    return static_cast<Message *>(RefCountedObject::get());
+  const Message *get() const {
+    return static_cast<const Message *>(RefCountedObject::get());
```

Comment:
I'm surprised this doesn't break the build but otherwise LGTM.

Reply:
It's fairly typical to have to `get()` methods, one `const` and the other not. That would probably be preferred.

Reply:
Yeah, I'm aware of that. I was just checking why the build succeeds. I'm reverting it anyway.